test-utils/store-validator/src/main.rs

```diff
@@ -0,0 +1,117 @@
+use std::path::Path;
+use std::process;
+use std::sync::Arc;
+
+use ansi_term::Color::{Green, Red};
+use clap::{App, Arg, SubCommand};
+
+use near_chain::{ChainStore, ChainStoreAccess};
+use near_logger_utils::init_integration_logger;
+use near_primitives::block::BlockHeader;
+use near_primitives::types::{BlockHeight, StateRoot};
+use near_store::{create_store, Store, StoreValidator};
+use neard::{get_default_home, get_store_path, load_config, NearConfig, NightshadeRuntime};
+
+#[allow(unused)]
+fn load_trie(
+    store: Arc<Store>,
+    home_dir: &Path,
+    near_config: &NearConfig,
+) -> (NightshadeRuntime, Vec<StateRoot>, BlockHeader) {
+    load_trie_stop_at_height(store, home_dir, near_config, None)
+}
+
+fn load_trie_stop_at_height(
```

Comment:
why did you copy this function?

Reply:
This part is WIP and will be used as soon as State tests appear.

Reply:
Okay. If the function is not used maybe there is no need to include it in the PR?

Reply:
removed